homeassistant/components/sensor/folder.py

```diff
@@ -29,10 +30,17 @@
 })
 
 
+def get_timestamp(file_path):
+    """Return the timestamp of file."""
+    mtime = os.stat(file_path).st_mtime
+    return datetime.datetime.fromtimestamp(mtime).isoformat()
```

Comment:
Not sure, but I think the helpers directory has a similar function somewhere. Can't check very well right now though because I'm on my phone.